src/deps.ts

```diff
@@ -90,9 +90,8 @@ const builtins = new Set([
   'zlib',
 ])
 
-const nameRe =
-  /^(?<name>(@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*)\/?.*$/i
```

Comment:
my tool doesn't support named groups (yet), so I switched it to a standard one, and then the [issue](https://redosdetector.com/?pattern=%5E%28%28%40%5Ba-z0-9-%7E%5D%5Ba-z0-9-._%7E%5D*%5C%2F%29%3F%5Ba-z0-9-%7E%5D%5Ba-z0-9-._%7E%5D*%29%5C%2F%3F.*%24&caseInsensitive=true&unicode=false) was the `.*` at the end overlapping with the rest.

The simplest solution looks like removing that and the end anchor.